android/src/main/java/com/tailscale/ipn/App.kt

```diff
@@ -62,22 +48,17 @@ class App : Application(), libtailscale.AppContext {
     const val STATUS_NOTIFICATION_ID = 1
     const val NOTIFY_CHANNEL_ID = "tailscale-notify"
     const val NOTIFY_NOTIFICATION_ID = 2
-    private const val PEER_TAG = "peer"
     private const val FILE_CHANNEL_ID = "tailscale-files"
     private const val FILE_NOTIFICATION_ID = 3
     private const val TAG = "App"
+
     private val networkConnectivityRequest =
         NetworkRequest.Builder()
             .addCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)
             .addCapability(NetworkCapabilities.NET_CAPABILITY_NOT_VPN)
             .build()
-    lateinit var appInstance: App
 
-    @JvmStatic
-    fun startActivityForResult(act: Activity, intent: Intent?, request: Int) {
```

Comment:
I think we still need this. We could definitely refactor this, but right now this launches Peer with the request VPN intent